### DISCUSSION TITLE
Global Settings

### DISCUSSION THREAD

**Discussion by nishaul** [filebrowser/filebrowser#2139] [Q&A]
Hello,

How to change Instance name width and height of FileBrowser page OR

how to change Font size of Instance name

**Comment by nishaul** [comment]
Is it not possible?

**Comment by ramiresviana** [comment]
The Custom Branding section of the documentation provides information about customizing FileBrowser, check https://filebrowser.org/configuration/custom-branding

**Comment by wassimux** [comment]
Hi, 
Did you try to customise css file?
